packages/vscode-extension/src/utilities/testCommand.ts

```diff
@@ -0,0 +1,15 @@
+import { command } from "./subprocess";
+
+export async function testCommand(cmd: string) {
```

Comment:
A doc comment and an explicit return type would be nice.
Previously this was used internally as part of the implementation, so you were assumed to know what it does while working on the code surrounding it, but it's useful to not have to look up the implementation of a util we use throughout the code base every time you encounter it.